### ISSUE TITLE
Support item parameter for CSV import

### ISSUE THREAD

**Issue by kenwenzel** [linkedfactory/linkedfactory-pod#16]
If a CSV file is uploaded then currently the headers need to specify the item and the property in the format
`<item IRI>@<property IRI>`

If the item is the same for all rows then it could be specified by using the HTTP parameter `item`.

@m-alreeni

**Comment by m-alreeni** [comment]
> If a CSV file is uploaded then currently the headers need to specify the item and the property in the format `<item IRI>@<property IRI>`
> 
> If the item is the same for all rows then it could be specified by using the HTTP parameter `item`.
> 
> @m-alreeni

yes this can be a next step (specifying  context and item as HTTP parameter). For testing purpose in some actaul projects, it is suffiecient for now to specify the item and the property in the format
<item IRI>@<property IRI>
